Brand-Consolidation - Burials Hub - populate link values

ncksllvn opened the pull request:
This PR fills in the links on the Burial Hub, and moves the page from `/burials/` to `/burials-memorials/` to be consistent with the rest of the website.

- Also contains work in this PR - department-of-veterans-affairs/vets-website/pull/8357
- Also fixes the layout for `/education/check-post-9-11-gi-bill-benefit-status/`

Related ticket - department-of-veterans-affairs/vets.gov-team/issues/12774

Commit:
Move /burials to /burials-memorials and update links

Commit:
FIx broken links

mnorthuis commented:
A couple of tweaks

- [ ] Request Military Records - This item should not link anywhere yet
- [ ] VA Survivor Rates - This link needs the "www" in front of it (users on VA network cannot access without it)
- [ ] Parents Dependency...Rates - Needs "www"
- [ ] Beneficiary Financial Counseling - Needs "www"
- [ ] Fudiciary Services - Needs - "www"

DanielleSoCompany committed:
Splitting link to survivor and dependents into 2

to align with new IA

DanielleSoCompany committed:
adding <br>

DanielleSoCompany commented:
Couple of things:

- [ ] A link within a page is going to the Survivors and Dependents page that we removed. I've updated it in this PR and requested a review from @bethpotts (always want a final editorial look on things before merging in case of typos/etc). Beth, can you please review and then request the merge from @ncksllvn? Thank you!
- [ ] Not sure what's going on on [this page](https://vetsgov-pr-8355.herokuapp.com/burials-memorials/veterans-burial-allowance/) or if it has anything to do with this specific work, but I noticed that instead of the "How do I apply?" copy and button, it's showing this error message:
![image](https://user-images.githubusercontent.com/22593175/45259298-7c33c000-b397-11e8-9273-0ea3afd7053f.png)

Commit:
Remove extra divs

Commit:
Merge remote-tracking branch 'origin/va-gov/changes-for-burials-review' into va-gov/burials-hub

Commit:
PR comments

ncksllvn commented:
> A link within a page is going to the Survivors and Dependents page that we removed. I've updated it in this PR and requested a review from @bethpotts (always want a final editorial look on things before merging in case of typos/etc). Beth, can you please review and then request the merge from @ncksllvn? Thank you!

I merged those changes into this PR since they're related.

ncksllvn commented:
> Not sure what's going on on this page or if it has anything to do with this specific work, but I noticed that instead of the "How do I apply?" copy and button, it's showing this error message:

That's powered by React, and is complicated because the React application code needs to be shared by Vets.gov. I've captured the work in this ticket - department-of-veterans-affairs/vets.gov-team/issues/12939 - so that we can go ahead and get this work merged in.

DanielleSoCompany commented:
Hi @ncksllvn. Reviewed the rest of the pages. Saw a few other things. Thanks!

**Related to URLS**
- [ ] This url (https://vetsgov-pr-8355.herokuapp.com/burials-memorials/what-to-expect-at-military-funeral/) is showing a blank page. Other pages that link to this page are linking to the old url (https://vetsgov-pr-8355.herokuapp.com/burials-and-memorials/what-to-expect-at-a-funeral/).

**Other**
- [ ] A few of the links/descriptions on the hub page seem like they don't have enough space around them
- [ ] The accordions don't seem to be opening on this page: https://vetsgov-pr-8355.herokuapp.com/burials-memorials/dependency-indemnity-compensation/

RobAWilkinson commented on va-gov/pages/burials-memorials/index.md:
```diff
@@ -1,55 +1,188 @@
+    <p>View current survivor benefit rates for parents.</p>
+
+  <div class="link">
+    <a href="https://www.archives.gov/veterans/replace-medals.html"><b>Replace Medals, Awards, and Decorations</b></a>
```
This is dead link, I think supposed to be: www.cem.va.gov/recmed_records.asp

RobAWilkinson replied:
nvm, that leads right back to that dead link 👎

Commit:
Remove double USWDS import

Commit:
Update references to Military Funeral page

Commit:
Delete burials-and-memorials directory

Commit:
Update references to /dependency-indemnity-compensation/

Commit:
Fix conflict with generated index file
